docs/UserGuide.md

```diff
@@ -2,41 +2,169 @@
 
-{Give steps to get started quickly}
-
 1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+2. Down the latest version of `tp.jar` from [here](https://github.com/AY2324S2-CS2113-T15-2/tp/releases/latest).
+3. Copy the file to the folder you want to use as the home folder for your BudgetBuddy app.
+4. Open a command terminal, `cd` into the folder you added the jar file to, and use the `java -jar tp.jar` command to
+   run the application.
+5. When prompted, type the command into the terminal and press Enter to execute it. e.g. typing `help` and pressing
+   Enter will prompt the help feature.
+
+## Features
+
+### Viewing help: `help`
+
+Shows the instructions for using BudgetBuddy.
+Format: `help`
+This command gives the list to search for commands, specific for each task. The commands `help all` and 
+`help acc` covers the basic structure of all commands. Instructions for command-specific help will be 
+provided in the `help all` and `help acc` accordingly.
+
+### Adding a transaction: `add`
+
+Adds a transaction into the transaction list of the specified account.
+Parameters: Account Number, Transaction Type, Name, Amount, Date, Category
+Format: `add /a/ACCOUNT_NUMBER /t/TRANSACTION_TYPE /n/NAME /$/AMOUNT /d/DATE /c/CATEGORY`
+
+* The `ACCOUNT_NUMBER` can be viewed using the command `list-acc`.
+* The `TRANSACTION_TYPE` includes **Expense** or **Income** ONLY.
+* The `AMOUNT` is in dollars ($).
+* The `DATE` should be in the format **DD-MM-YYYY**.
+* The `CATEGORY` is an integer. The categories are mapped to the following integers: 1 (Dining), 2 (Groceries), 3 (
+  Utilities), 4 (Transportation), 5 (Healthcare), 6 (Entertainment), 7 (Rent), 8 (Salary), 9 (Others).
+* NOTE: A transaction of a particular `CATEGORY` can be of both `TRANSACTION_TYPE`s.
+
+Example of usage:
+
+`add /a/5431 /t/Income /n/March Salary /$/10000 /d/01-03-2024 /c/8`
+
+`add /n/New iPhone /$/2000 /c/9 /t/Expense /a/5431 /d/20-03-2024`
+
+### View transaction history: `list`
+
+List the transactions. List options will include 1. All Transactions, 2. Past Week Transactions, 3. Past Month
+Transactions, 4. Custom Date Transactions.
+Format: `list`
+
+Example of usage:
+`list` followed by `1` to view All Transactions.
+`list` followed by `2` to view Past Week Transactions.
+`list` followed by `3` to view Past Week Transactions.
+`list` followed by `4` followed by the start date `01-01-2024` followed by the end date `31-03-2024` to view Custom Date
+Transactions from 01-01-2024 to 31-03-2024.
+
+### Deleting a transaction: `delete`
+
+Removes a transaction from transaction history.
+Parameters: Transaction ID
+Format: `delete TRANSACTION_ID`
+
+* The `TRANSACTION_ID` is an integer value ranges from one to the size of the transaction history (index
+  ID of the last transaction).
+* The `TRANSACTION_ID` can be viewed using the command `list` followed by `1`.
+
+Example of usage:
+`delete 1`
```

Comment:
The expected output can also be included for users to more clearly know what to expect when using the programme.